source/controllers/User.ts

```diff
@@ -70,11 +96,24 @@ login.create = login.handler(
 	}
 );
 
+const signout = user.subcontroller('signout');
+
+signout.read = async (req, res) => {
+	if (res.locals.user !== null && res.locals.user !== undefined) {
+		res.clearCookie('AuthToken');
+		res.redirect('/section');
+	} else {
+		res.send('Not Logged in');
+		res.redirect('/section');
+	}
+};
+
```

Comment:
I don't like this because reads should not delete data.
It violates the controller principles.
In my opinion it should be the login controller's delete method.
It does invite some JS fuckery into the client side because you need to add an onclick event on the sign out button but i think that would be the correct way to do it